src/main/java/no/nav/fo/veilarbaktivitet/service/AktivitetService.java

```diff
@@ -262,18 +260,14 @@ private StillingsoekAktivitetData mergeStillingSok(StillingsoekAktivitetData ori
                 .withStillingsTittel(stillingsoekAktivitetData.getStillingsTittel());
     }
 
-    private void lagreAktivitet(AktivitetData aktivitetData) {
-        try {
-            aktivitetDAO.insertAktivitet(aktivitetData);
-            if (unleash.isEnabled("veilarbaktivitet.kafka")) {
-                kafkaService.sendMelding(of(aktivitetData));
-            }
-        } catch (DuplicateKeyException e) {
```

Comment:
Trenger fremdeles dette!

Reply:
DuplicateKeyException ?

Reply:
Versjonskonflikt exception. Den må bli returnert til aktivitetsplan

Reply:
Du kan vel bara endre i aktivitetsplan? Versjonskonflikt er i tillegg depricated ;)

Reply:
is it?

Reply:
Huh, det stemmer. Dette burde endres, men jeg foreslår vi gjør dette senere, da vi må endre 2 systemer samtidig for at brukerne våre ikke skal oppleve noe rart

Reply:
....

Reply:
Førnøjd???